### PR TITLE
Cleanup tidy warnings and get more tests running.

### PR DESCRIPTION
**Pull request by jplevyak**
Signed-off-by: John Plevyak <jplevyak@gmail.com>

### CODE REVIEW

File: bazel/repositories.bzl [hunk 1]
```diff
@@ -743,8 +743,9 @@ def _proxy_wasm_cpp_sdk():
     _repository_impl(name = "proxy_wasm_cpp_sdk")
 
 def _proxy_wasm_cpp_host():
-    _repository_impl(
+    native.new_local_repository(
         name = "proxy_wasm_cpp_host",
+        path = "/home/jplevyak/work/proxy-wasm-cpp-host",
```
Comment (kyessenov) [inline bazel/repositories.bzl:748]: leftover?

Comment (jplevyak) [reply bazel/repositories.bzl]: reverted.

File: source/extensions/common/wasm/wasm.cc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #include "extensions/common/wasm/wasm.h"
 
-#include <stdio.h>
+#include <cstdio>
```
Comment (kyessenov) [inline source/extensions/common/wasm/wasm.cc:3]: Why do we need stdio here, do we print stdout directly?

Comment (jplevyak) [reply source/extensions/common/wasm/wasm.cc]: These are left over from debugging.  YCM doesn't seem to report unnecessary includes, but I checked and I can remove them.

**Pull request merged**